openlibrary/plugins/upstream/account.py

```diff
@@ -270,6 +271,50 @@ def GET(self):
         return render.account(user)
 
 
+@dataclass
+class PDOption:
+    """Represents an option in the print-disability qualifying organization selector"""
+
+    label: str
+    value: str
+
+
+def get_pd_options():
+    options = [
+        PDOption("BARD", "ia_nlsbardaccess_disabilityresources"),
+        PDOption("BookShare", "ia_bookshareaccess_disabilityresources"),
+        PDOption("ACE", "aceportalocul_disabilityresources"),
+        PDOption("I don't have one yet", "unqualified"),
+    ]
+
+    pd_orgs = cached_pd_org_query()
+    options += [PDOption(org.get("title"), org.get("identifier")) for org in pd_orgs]
+
+    return options
+
+
+def make_pd_org_query():
+    base_url = config.get("pda_org_search_url", "")
+    if not base_url:
+        return []
+    params = "q=collection:print_disability_access&fl[]=identifier,title&rows=1000&page=1&output=json"
```

Comment:
When ran yesterday, this query returned 502 qualifying organizations.  This query limits the results to `1000`.  It may be a long time until the `print_disability_access` collection has that many items, and it likely won't be immediately obvious to maintainers if this does occur.

How can we ensure that all qualifying organizations are included in the selector when the `print_disability_access` collection has more than 1,000 items?

Reply:
We can use the scrape API moving forward to fetch up to 10k results but will then have to re-work the dropdown probably. We haven't hit the 1k limit in 5 years and some orgs may actually drop off, so let's see what happens